### PR TITLE
Pr for landing page

### PR DESCRIPTION
**Pull request by suryxks**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,29 +9,55 @@
...
+            <p class="text-md">Nectar is a CSS library with pre-defined styled classes and Utilities to get you quickly
+                started with your project</p>
+            <div class="actions-hero">
+                <a href="./components/alert/alert.html"><button class="btn-cta">Docs</button></a>
+                <a href="https://github.com/suryxks/Nectar-ui" target="_blank"><button
+                        class="btn-cta-outlined">Github</button></a>
+            </div>
+        </div>
+        <img src="./assets/hero.svg" class="hero-img">
+
```
Comment (poornima-alugubelly) [inline index.html:42]: You could add an alt tag for img for better accessibility

File: index.html [hunk 2]
```diff
@@ -9,29 +9,55 @@
...
+
+    </section>
+    <div class="installation display">
+        <div>
+            <h1 class="h-1">Installation</h1>
+            <p class="text-md">Copy and paste the code in the head tag of your html document. So that you can style your
```
Comment (poornima-alugubelly) [inline index.html:48]: Can we create more descriptive classes instead of "h-1" say "large heading"

**Review by poornima-alugubelly — APPROVED** [review]

**Review by gopalbbharadva — APPROVED** [review]

File: style.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+.header-nav {
+    box-shadow: 0px 10px 4px -4px hsl(0deg 0% 0% / 0.2);
+    position: sticky;
+    top:0;
+
+}
+
+.hero {
+
```
Comment (msdeshmukh009) [inline style.css:12]: Use some code formatter like prettier that will take care of these kinds of spaces.

File: index.html [hunk 4]
```diff
@@ -9,29 +9,55 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
```
Comment (msdeshmukh009) [inline index.html]: Change the title to your CL's name.

**Review by msdeshmukh009 — APPROVED** [review]
Looks good to go.

File: index.html (hunk 4 above)
Comment (suryxks) [reply index.html]: Thank you for pointing it out mahesh

File: index.html (hunk 2 above)
Comment (suryxks) [reply index.html:48]: Thank you for the feedback poornima ,i have made the changes and will merge the PR

**Pull request merged**